tests/unit/test_config.py

```diff
@@ -23,6 +23,7 @@
 from cylc.flow.config import SuiteConfig
 from cylc.flow.cycling import loader
 from cylc.flow.exceptions import SuiteConfigError
+from cylc.flow.tests.conftest import mock_glbl_cfg
```

Comment:
```suggestion
from tests.unit.conftest import mock_glbl_cfg
```

Reply:
This should fix the test_config GH actions failure. 
I also have a failure locally on the integration tests that can be fixed by adding a  `        'platform': {'name':'platform'},
` into the job conf in tests/integration/test_job_pool.py.